ubuntu-github-actions--24.04/Dockerfile

```diff
@@ -55,7 +55,6 @@ RUN \
     pkg-config \
     && \
   apt-get clean && \
-  python3 -m pip install --upgrade --break-system-packages pip && \
```

Comment:
Why does this line need to be deleted?

Reply:
Pip cannot remove system level pip instance.